packages/web-components/src/components/leaving-ibm/leaving-ibm-modal-heading.ts

```diff
@@ -21,6 +21,13 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-leaving-ibm-modal-heading`)
 class DDSLeavingIbmModalHeading extends BXModalHeading {
+  connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'heading');
+    }
+    super.connectedCallback();
+  }
```

Comment:
Could you add `aria-level`, too, in a similar way to [`<dds-card-heading>`](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.15.0/packages/web-components/src/components/card/card-heading.ts#L32-L34)?